ynput/ayon-zbrush
# Remove zbrush menu which is generated by the communication server during previous testing

## Changelog Description
This PR is to remove unnecessary zbrush menu zscript which is generated from the server during the testing of the previous PR.


## Additional info
n/a


## Testing notes:

1. Launch Zbrush via Ayon
2. Check if the menu here
3. Close Zbrush
4. The zscript menu from should be removed as expected.
